shared/src/store/action-reducers/transfer-point.ts

```diff
@@ -141,8 +143,13 @@ export namespace TransferPointActionReducers {
                     'transferPoints',
                     transferPointId
                 );
-                transferPoint.internalName = internalName;
-                transferPoint.externalName = externalName;
+                // Empty strings are ignored
```

Comment:
This is a breaking change. Could this break something? Probably not. Do we want to risk it?